### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by mateo951**
- Added API functionality
- Modified demo file
- Added fix for linters

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/displayData.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  listItems.innerHTML = '';
+  result.forEach((item) => {
+    listItems.innerHTML += `<li class="list-group-item">${item.user}: <span class="floatRight badge badge-primary badge-pill">${item.score}</span></li>`;
+  });
+
+//   list.innerHTML = '';
+//   scores.forEach((score) => {
+//     list.innerHTML += `<li>${score[0]}: ${score[1]}</li>`;
+//   });
+//   if (scores.length > 5) { list.classList.add('scroll'); }
```
Comment (mikenath223) [inline src/modules/displayData.js]: - [x] Kindly remove this commented code. It makes the project quite untidy. Refer to GitHistory if you want to reuse previous code.

Comment (mateo951) [reply src/modules/displayData.js]: Thanks a lot for the heads-up! Changed applied 😎

**Review by george-shammar — APPROVED** [review]
Hi @mateo951 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**